src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const button = body.querySelector('.logo');
+
+function createPromise() {
+  return new Promise((resolve, reject) => {
+    button.onclick = () => {
```

Comment:
```suggestion
    button.addEventListener('click', () => {})
```
use addEventListener instead of onclick